packages/feedback/src/app/modules/feedback/feedback.module.ts

```diff
@@ -0,0 +1,28 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
+import {NgPipesModule} from 'ngx-pipes';
+import { NgxPaginationModule } from 'ngx-pagination';
+import { NgXtruncateModule } from 'ngx-truncate';
+import { FeedbackHomeComponent } from './home/feedback-home.component';
+import {FeedbackItemComponent} from './feedback-item/feedback-item.component';
+import {FeedbackService} from './feedback.service'; 
+
+@NgModule({
+  declarations: [ 
+    FeedbackHomeComponent,
+    FeedbackItemComponent,
+  ],
+  providers: [
+    FeedbackService,
+  ],
+  imports: [
+    CommonModule,
+    FormsModule,
+    NgPipesModule,
+    NgxPaginationModule,
+    NgXtruncateModule,
+
+  ]
+})
+export class FeedbackModule { }
```

Comment:
As I said above, there's no need for a sub-module for Feedback, as the entire entire application is for Feedback.

A sub-module should only be used if the application is very complex and needs multiple sub modules.

Reply:
Done.